templates/prometheus.yaml

```
@@ -5,6 +5,11 @@ metadata:
   namespace: kubeaddons
   labels:
     kubeaddons.mesosphere.io/name: prometheus
+  annotations:
+    kubeaddons.mesosphere.io/appVersion: "0.31.1"
+    kubeaddons.mesosphere.io/endpoint-prometheus: "/ops/portal/prometheus"
```

Let's use a slash instead of a dash for all the endpoints:
```suggestion
    kubeaddons.mesosphere.io/endpoint/prometheus: "/ops/portal/prometheus"
```